mne/decoding/tests/test_base.py

```diff
@@ -114,16 +133,43 @@ def inverse_transform(self, X):
         invs = _get_inverse_funcs(est)
         assert_equal(invs, list())
 
-    # II. Test get coef for simple estimator and pipelines
-    for clf in (lm, make_pipeline(StandardScaler(), lm)):
+    # II. Test get coef for classification/regression estimators and pipelines
+    for clf in (lm_regression,
+                lm_gs_classification,
+                make_pipeline(StandardScaler(), lm_classification),
+                make_pipeline(StandardScaler(), lm_gs_regression)):
+
+        # generate some categorical/continuous data
+        # according to the type of estimator.
+        if is_classifier(clf):
+            n, n_features = 1000, 3
+            X = np.random.rand(n, n_features)
```

Comment:
Please use a random state to this test has no effect on global state

Reply:
Sorry for the late replay.
Does this mean that it would be better to use the `np.random.seed(123)` or `np.random.RandomState(123)` before the `np.random.rand()` function?

Reply:
Typically we try to do:
```
def test_whatever():
    rng = np.random.RandomState(0)
    ...
       X = rng.rand(n, n_features)
```

Reply:
Oh! I see~ Let me modify it.